style.css

```diff
@@ -0,0 +1,1280 @@
+    width: 29px;
+    height: 4px;
+    margin: 5px 24px;
+  }
+
+  .wrap {
+    margin-bottom: 32px;
+  }
+
+  .wrap__img {
+    padding-left: 155px;
+  }
+
+  .wrap__main-img {
+    height: 432px;
+    margin: 0 43px 56px 0;
+  }
+
+  .wrap__heading {
+    font-size: 67px;
+    line-height: 68px;
+    letter-spacing: -.2px;
+    margin-bottom: 18px;
+  }
+
+  .wrap__article {
+    line-height: 17px;
+    width: 390px;
+    letter-spacing: .3px;
+    margin-bottom: 33px;
+    font-size: 12px;
+  }
+
+  .wrap__button-next {
+    margin-right: 45px;
+    height: 37px;
+    padding: 3px 32px;
+    border: 1px solid #000;
+    line-height: 27px;
+    font-size: 12px;
+  }
+
+  .wrap__button-more {
+    font-size: 11px;
+    letter-spacing: 0;
+  }
+
+  .button-more__vector {
+    padding-left: 12px;
+    width: 35px;
+  }
+
+  .footer__social-networks {
+    padding-left: 44px;
+  }
+
+  .footer__social-button {
+    width: 46px;
+  }
+
+  .footer__scroll {
+    width: 24px;
+    height: 34px;
+  }
+}
+
+@media (max-width: 1100px) {
+  .first-page {
+    background: linear-gradient(to right, #000 352px, #fff 352px);
+    min-height: 688px;
+  }
+
+  .header {
+    height: 71px;
+    margin-bottom: 58px;
+  }
+
+  .header__logo {
+    padding: 27px 0 0 40px;
+  }
+
+  .header__logo-loco {
+    font-size: 23px;
+    margin-right: 185px;
+    letter-spacing: 1px;
+    font-weight: 600;
+  }
+
+  .header__lang {
+    padding-top: 4px;
+  }
+
+  .header__lang-ru {
+    font-size: 14px;
+  }
+
+  .lang__vector {
+    width: 11px;
+    height: 5px;
+  }
+
+  .header__menu {
+    height: 70px;
+    width: 70px;
+  }
+
+  .header__menu-line {
+    width: 26px;
+    height: 3px;
+    margin: 6px 22px;
+  }
+
+  .header__menu-icon {
+    height: 28px;
+  }
+
+  .wrap {
+    margin-bottom: 24px;
+  }
+
+  .wrap__main-img {
+    height: 398px;
+    margin: 0 38px 56px 0;
+  }
+
+  .wrap__img {
+    padding-left: 142px;
+  }
+
+  .wrap__heading {
+    font-size: 61px;
+    line-height: 63px;
+    letter-spacing: 0;
+    margin-bottom: 14px;
+  }
+
+  .wrap__article {
+    line-height: 16px;
+    width: 350px;
+    letter-spacing: .6px;
+    margin-bottom: 29px;
+    font-size: 11px;
+  }
+
+  .wrap__button-next {
+    height: 33px;
+    padding: 3px 31px;
+    line-height: 23px;
+    font-size: 10px;
+    margin-right: 43px;
+  }
+
+  .wrap__button-more {
+    font-size: 10px;
+    letter-spacing: -.5px;
+  }
+
+  .button-more__vector {
+    padding-left: 13px;
+    width: 33px;
+  }
+
+  .footer__social-networks {
+    padding-left: 40px;
+  }
+
+  .footer__social-button {
+    width: 42px;
+  }
+
+  .social__item-inst {
+    width: 78%;
+  }
+
+  .social__item-vk {
+    width: 78%;
+  }
+
+  .social__item-fb {
+    width: 78%;
+  }
+
+  .footer__scroll {
+    width: 23px;
+    height: 30px;
+  }
+}
+
+@media (max-width: 992px) { 
+  .first-page {
+    background: linear-gradient(to right, #000 318px, #fff 318px);
+    min-height: 620px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633354)